drivers/mbox/mbox_nrf_vevif_task_tx.c

```diff
@@ -3,31 +3,37 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define DT_DRV_COMPAT nordic_nrf_vevif_remote
+#define DT_DRV_COMPAT nordic_nrf_vevif_task_tx
```

Comment:
Shouldn't it be rx? If I understand correctly, bellboard is used to trigger IRQ on other core from VPR, so "remote" is equivalent to tx, but vevif task is used to trigger IRQ on VPR from other core, so, in this case, "remote" equivalent should be rx. Or am I missing something?

Reply:
The earlier nomenclature may have been confusing.

Currently we have `tx` and `rx` depending on the purpose of a specific node.
If we use `vevif events`, the signal goes from the VPR core to the remote core.
So, we have the `vevif_event_tx` node, which we set up in the VPR dts and as the name suggests, we use to transmit the signals (tx) from the VPR core to the remote core.
On the other hand, we have the `vevif_event_rx` node, which we configure in the remote core dts and use to receive the signals (rx) on the remote core.

Similarly, we have `vevif tasks` that are used for signaling from the remote core to the VPR core.
So, we have the `vevif_task_rx` node, which we set up in the VPR dts and as the name suggests, we use to receive the signals (rx) on the VPR core.
On the other hand, we have the `vevif_task_tx` node, which we configure in the remote core dts and use to transmit the signals (tx) from the remote core to the VPR core.
It seems quite analogous to me.

Reply:
Ok, I understand now, I thought that "remote" always refers to RISCV core and "local" always refers to other core (e.g. ARM). I did not know that "remote" means core that triggers IRQs.